### PR TITLE
ENT-4807 Switch to cleaning up records relative from the most recent record of that type

### PR DESCRIPTION
**Pull request by nickanderson**

### CODE REVIEW

**Comment by cf-bottom** [comment]
Thanks for submitting a PR! Maybe @craigcomstock can review this?

**Comment by mikeweilgart** [comment]
(I wouldn't mind some credit for the suggestion.)  :)

File: lib/cfe_internal_hub.cf [hunk 1]
```diff
@@ -73,65 +73,67 @@ bundle agent cfe_internal_hub_maintain
...
-                                                     {
-                                                         "report": "diagnostics",
-                                                         "table": "Diagnostics",
-                                                         "history_length_days": 1,
-                                                         "time_key": "TimeStamp"
-                                                     }
-                                                    ]');
+      "report_settings"
+        data => parsejson('[
+  {
```
Comment (craigcomstock) [inline lib/cfe_internal_hub.cf]: why all the whitespace change here?

Comment (nickanderson) [reply lib/cfe_internal_hub.cf]: readability in terminals that are limited to 80 chars!

**Pull request merged**